### PR TITLE
My Goa Cheatsheet

### PR DESCRIPTION
**Pull request by tanvimirashi18**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

What changes does this PR introduce? 

###### Related Issues or Discussions 

What issues or discussions (if any) does this PR address? Please provide the issue # or link.

###### People to notify

@mention any relevant people/organizations

---

Instant Answer Page: https://duck.co/ia/view/{{ID}}

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mention

### CODE REVIEW

**Comment by daxtheduck** [comment]
@tanvimirashi18 Thanks for the pull request! Unfortunately, we're only accepting contributions and improvements to a limited portion of the search space.
            
The staff and community are currently focused on making DuckDuckGo the [best search engine for programmers](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53). To ensure we're making as much progress as possible, **we are only accepting Instant Answers and updates related to the programming mission**.
    
If this is not related to the Programming Mission, it will likely be closed but you can:
    
1. Help us prevent further confusion by telling us where you found out about DuckDuckHack or where we could've alerted you earlier.
2. Keep this code handy! We'll be expanding into other search topics in the future!
3. Join us at our new home: https://forum.duckduckhack.com/

If this is related to the Programming Mission, it will be reviewed by staff and the community as soon as possible!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/language/goa.json [hunk 1]
```diff
@@ -0,0 +1,202 @@
+{  
+   "template_type":"reference",
+   "id":"goa_cheat_sheet",
+   "name":"goa",
+   "description":"Goa is a state in western India with coastlines stretching along the Arabian Sea. Its long history as a Portuguese colony prior to 1961 is evident in its preserved 17th-century churches and the area’s tropical spice plantations. Goa is also known for its beaches, ranging from popular stretches at Baga and Palolem to those in laid-back fishing villages such as Agonda.",
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/language/goa.json]: make every keys and descriptions shorter

**Comment by gautamkrishnar** [comment]
I don't think that this one can be accepted. Sorry, we have certain regulations on cheatsheets. It must show concise info about a specific topic. Feel free to open a pull request after making the cheatsheet on some other topics...
cc: @GuiltyDolphin

**Comment by sahildua2305** [comment]
@tanvimirashi18 sorry, we can't accept this type of cheat sheets. The purpose of a cheat sheet is to provide concise information to users. This cheat sheet violates the mentioned purpose.

While, I will be closing this PR, please feel free to create cheat sheets for other topics which you feel is missing. Here's how you can help us with our mission to improve programming results - https://forum.duckduckhack.com/c/programming

**Pull request closed without merging**